app/views/channels.html

```diff
@@ -49,6 +49,7 @@
                   <td>
 
                     <div ng-if="channel.status!='deleted'">
+                      <button class="btn btn-info btn-xs" ng-click="duplicateChannel(channel)"><i class="glyphicon glyphicon-file"></i></button>
```

Comment:
Ok I see. I think tooltips would be ok. Even if they just say 'duplicate', 'edit' and 'delete'. I'm not such a fan of adding these to the header, it's not very clear. If you could provide a screenshot of what you think looks bad we could try iterate on that.